tests/foreman/cli/test_oscap.py

```diff
@@ -104,8 +104,10 @@ def setUpClass(cls):
             remote_file="/tmp/{0}".format(cls.file_name)
         )
         cls.title = 'rhel-6-content'
-        result = Scapcontent.info({'title': cls.title})
-        if not result['title'] in cls.title:
+        result = Scapcontent.list()
```

Comment:
No need to initiate a result variable here. Just directly use ```Scapcontent.list()``` below.

Reply:
need the output of cmd in variable

Reply:
Addressed this.